motoko/basic_bitcoin/dfx.json

```
@@ -0,0 +1,32 @@
+{
+  "version": 1,
+  "dfx": "0.10.2-btcbeta.0",
```

A note so that we don't forget: Before we publish the example, we'll want to have the proper version of the SDK published, and update the version here.

---

I hesitated as I am not keen with SDK versions, what is new in `0.9.1-211-g72cb0295` that isn't in `0.10.2-btcbeta.0` ? and how to get the version `0.9.1-211-g72cb0295` of DFX ?

```suggestion
  "dfx": "0.9.1-211-g72cb0295",
```

---

In fact it seems that with `0.9.1-224-gec4e0615` I have the error `[Canister r7inp-6aaaa-aaaaa-aaabq-cai] Panicked at 'called `Result::unwrap()` on an `Err` value: (CanisterReject, "The bitcoin_get_balance API supports only the Testnet network.")', src/basic_bitcoin/src/bitcoin_api.rs:53:15` while I don't have it with `0.9.1-224-gec4e0615`.

---

```suggestion
  "dfx": "0.11.0-beta.1",
```
I tested [this new version](https://github.com/dfinity/sdk/releases/tag/0.11.0-beta.1) on my side and it works fine.